app/src/main/java/com/duckduckgo/app/notification/NotificationScheduler.kt

```diff
@@ -72,7 +68,7 @@ class NotificationScheduler @Inject constructor(
 
     // Legacy code. Unused class required for users who already have this notification scheduled from previous version. We can delete this in a
     // couple of weeks once old notifications have cleared. We should also remove "open" access from ClearDataNotificationWorker.
-    class ShowClearDataNotification(context: Context, params: WorkerParameters): ClearDataNotificationWorker(context, params)
+    class ShowClearDataNotification(context: Context, params: WorkerParameters) : ClearDataNotificationWorker(context, params)
```

Comment:
when has "a couple of weeks" passed? do we have a task scheduled for this cleanup?

Reply:
That's well and truly passed. No need for a follow up, I'll delete it now.

Reply:
Oh, actually, maybe we should do it at the same time as https://app.asana.com/0/414730916066338/1119619712088571?

So either leave it be until that's done, or do that task while you're removing it?

Reply:
I did think about this but then figured that this notification shouldn't exist at all anymore unless some user newly installs the version that contained it. Stranger things have happened though and users may reset their app data and mimic a fresh install. I've left it in but updated the comment and the asana task. I think that's safest. Good shout!